README.md

```diff
@@ -116,19 +116,38 @@ It is important for any code change to pass both unit and end-to-end tests. This
 
 ### Run End-to-End (E2E) Tests
 
-1. Run the token bridge UI locally on `http://localhost:3000/`
+1. Set up the Nitro node
+
+   1. Download and install [Docker](https://www.docker.com/)
 
-2. Set env vars (one time):
+   2. Check out the [Nitro repo](https://github.com/OffchainLabs/nitro)
 
-   1. At this folder's root,
+   3. Run: (Make sure your Docker App is running)
 
       ```bash
-      $ cp ./packages/arb-token-bridge-ui/.e2e.env.sample ./packages/arb-token-bridge-ui/.e2e.env
+      $ ./test-node.bash --init --no-blockscout
```

Comment:
https://github.com/OffchainLabs/nitro/pull/1588
since this PR we no longer need to use the `--no-blockscout` flag as it's disabled by default